mimic/canned_responses/fastly.py

```diff
@@ -0,0 +1,278 @@
+"""
+Canned response for fastly
+"""
+import uuid
+
+
+class FastlyResponse(object):
+
+    """
+    Canned response for fastly.
+    See docs here https://docs.fastly.com/api/config
+    """
+
+    fastly_cache = {}
+
+    def get_current_customer(self):
+        """
+        Returns the current customer with response code 200.
+
+        :return: a JSON-serializable dictionary matching the format of the JSON
+                 response for fastly_client.get_current_customer()
+                 ("/current_customer") request.
+        """
+        current_customer = {
```

Comment:
This whole dictionary is built without any reference to `self`, so defining it outside - as a class variable, say - would make it clearer that we are not actually building anything based on the current object just yet.  As it was I had to carefully look through to see if `self` was being used.